zipkin-lens/src/test/data/malformed.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "traceId": "6cf5624e3452d1bc202a57dad35921c0",
+    "parentId": "d65495313f944801",
+    "id": "d5cb70daead478dc",
+    "name": "kong.header_filter",
+    "timestamp": 1570005203864000
```

Comment:
this should have been an annotation

Reply:
I think what's missing here is that a span can include annotations.. annotations have a required timestamp. So instead of making a sub-span that only includes a name and a timestamp, make an annotation with that name and timestamp.

ex 
```json
{
    "traceId": "6cf5624e3452d1bc202a57dad35921c0",
    "parentId": "ed395cb866d02665",
    "id": "d65495313f944801",
    "kind": "CLIENT",
    "name": "get /users/{userId}",
    "timestamp": 1570005203829000,
    "duration": 35000,
--snip--
    "annotations": [
       {"name": "kong.header_filter", "timestamp": 1570005203864000}
     ]
```